packages/bottender-express/src/createServer.ts

```diff
@@ -7,7 +7,15 @@ import { Bot, RouteConfig } from './types';
 function createServer(bot: Bot, config: RouteConfig = {}) {
   const server = express();
 
-  server.use(bodyParser.urlencoded({ extended: false }));
+  server.use(
```

Comment:
Should we add some comments on top of this to show that those lines are   written for Slack?

Reply:
Anyway, I'm going to merge this. We can put the comments later on.